### DISCUSSION TITLE
Paired analysis of differential expression

### DISCUSSION THREAD

**Discussion by benjacobs123456** [satijalab/seurat#5150] [Q&A]
Hi there

Sorry if this is a silly question. Is there a neat way within Seurat of analysing differential expression between two states / tissues taking into account that the cells come from the same donor?

To spell this out - say I have single cell RNA seq data from blood and kidney of a few donors. I am interested in the changes in gene expression between, say, CD4 T cells in blood and CD4 T cells in kidney. As far as I can tell, running FindMarkers on my combined dataset for the CD4 T cell cluster (multiple donors, 2 tissues) will compare bulk expression between the tissues without taking into account the paired samples. This may well be interesting, but I'm sure I lose interesting within-individual compartment-specific changes in gene expression by analysing en masse. 

I know I can simply split the dataset into individual donors and do DE testing for each donor individually. This may be the best solution, but I was wondering if there's any neater built-in functionality to do this in the merged dataset.

Cheers
Ben

**Comment by benjacobs123456** [comment]
I guess maybe using donor ID as a latent variable and regressing it out achieves the same thing...

**Comment by cswoboda** [comment]
Normally I downsample from each replicate (in your case donor) and create a merged object where I'm comparing equal # of cells across both conditions and both donors, but all in one run.
